.changeset/config.json

```diff
@@ -12,8 +12,7 @@
     "@local/*",
     "@tests/*",
     "error-stack",
-    "deer",
-    "antsi"
+    "deer"
```

Comment:
is this change related with this PR?

Reply:
Sorry, it was discussed [here ](https://hashintel.slack.com/archives/C022217GAHF/p1683393353949689)